docs/terms/execution_engine.md

```diff
@@ -58,10 +64,57 @@ You will not need to directly access an Execution Engine.  Instead, you will con
 
```

Comment:
I could potentially see a note here to the effect of "if you are interested in using an accessing data with an ExecutionEngine which we do not yet support, you may {consider a contribution, get in touch with us on Slack}. This is a big undertaking, and we are happy to offer guidance and support."

I don't exactly know what the right message here is. We've talked, for instance, about a new Execution Engine for something like Dask, and we have some outstanding work on a filesystem execution engine which can be used for expectations like `expect_file_to_exist`. The former would likely be a lot of work; the latter isn't unreasonable. And I think that the bulk of our Spark implementation was a community contribution. 

I could also imagine that this could be overwhelming in a doc like this, but wanted to raise this as a possibility.

Reply:
I think it's appropriate for open source documentation, and @kyleaton  agreed.  I've added the note.